sound/soc/sof/pcm.c

```diff
@@ -670,6 +670,11 @@ static int sof_pcm_dai_link_fixup(struct snd_soc_pcm_runtime *rtd,
 		return -EINVAL;
 	}
 
+	if (!dai->dai_config) {
+		dev_dbg(sdev->dev, "no DAI config for %s!\n", rtd->dai_link->name);
+		return 0;
+	}
+
```

Comment:
separate patch please

Reply:
@ranj063 as replied to @lgirdwood this only can happen for VirtIO front ends, don't think making this a separate patch would make much sense.